src/torchmetrics/image/lpip.py

```diff
@@ -137,7 +137,7 @@ def __init__(
     def update(self, img1: Tensor, img2: Tensor) -> None:
         """Update internal states with lpips score."""
         loss, total = _lpips_update(img1, img2, net=self.net, normalize=self.normalize)
-        self.sum_scores += loss.sum()
+        self.sum_scores += loss.sum().detach()
```

Comment:
Btw, we say the metric is differentiable. If we detach the tensor, we cannot call `backward` on that tensor. Is that something what we want?
cc: @Borda @SkafteNicki

Reply:
Maybe a detached version is needed if users do not care about the backward? If so, add a parameter of update to determine detach or not.